README.md

```diff
@@ -48,81 +54,70 @@ Don't forget to run `terraform init` and you're ready to go! 🚀
 
 Start browsing the [registry docs](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs) to find resources and data sources to use.
 
+## Roadmap
```

Comment:
Can you add it to the table of contents at the top ?

Reply:
Yes.

Reply:
Added.